src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.Windows.cs

```diff
@@ -217,7 +217,7 @@ private static bool GetSsl3Support()
                     enabled = false;
                 }
             }
-            catch (Exception e) when (e is SecurityException || e is InvalidCastException)
+            catch (Exception e) when (e is SecurityException || e is InvalidCastException || e is NullReferenceException)
```

Comment:
What throws a NullReferenceException?

Reply:
I did more testing and it seems like Registry.GetValue() can return null if the whole key is missing. With that, the (int) cast throws.